--- a/BUILD
+++ b/BUILD
@@ -56,7 +56,14 @@
 COMMON_CXXOPTS = ["-std=c++17"] + PLATFORM_COPTS
 
 COMMON_LINKOPTS = select({
-    "@platforms//os:windows": [],
+    "@platforms//os:windows": [
+        "-DEFAULTLIB:advapi32.lib",
+        "-DEFAULTLIB:dbghelp.lib",
+        "-DEFAULTLIB:shlwapi.lib",
+        "-DEFAULTLIB:ws2_32.lib",
+        "-DEFAULTLIB:ole32.lib",
+        "-DEFAULTLIB:shell32.lib",
+    ],
     "//conditions:default": ["-lm"],
 })
 
